x-pack/plugins/apm/server/routes/mobile/get_mobile_stats.ts

```diff
@@ -74,6 +74,7 @@ export async function getMobileStats({
         ProcessorEvent.error,
         ProcessorEvent.metric,
         ProcessorEvent.transaction,
+        ProcessorEvent.span,
```

Comment:
Do you need to query all those indices ti retrieve mobile stats? Eg. do we expect the dimensions to be available on metrics and errors?

Reply:
as we discussed, I will fix the indices for the mobile endpoints in a follow up PR